operator/src/main/java/org/projectnessie/operator/resource/NessieImage.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2024 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.operator.resource;
+
+public class NessieImage {
+
+  public enum PullPolicy {
+    Always,
+    Never,
+    IfNotPresent
+  }
+
+  private PullPolicy pullPolicy = PullPolicy.IfNotPresent;
```

Comment:
Should this default to `Always` for `latest` (and `latest-java`) and `IfNotPresent` for other tags?

Reply:
OK, well, we need to "split" the image name then :-)